### PR TITLE
Reduce some codacy issues #2

### PR DESCRIPTION
**Pull request by darkwizz**
To #41

### CODE REVIEW

File: app/src/main/java/in/aviaryan/hinix/GameBoard.java [hunk 1]
```diff
@@ -39,6 +39,78 @@ public GameBoard(InputStream wordListStream) throws IOException {
...
+                }
+                if (shortRun == 1){
+                    shortRun = 2; i=0; j=-1;
+                }
+            }
+        }
+        return false;
+    }
+
+    private boolean hasBadLetters(String word, String badLetters) {
```
Comment (aviaryan) [inline app/src/main/java/in/aviaryan/hinix/GameBoard.java]: Can you please move badLetters inside this function and not pass it as a parameter. Or better, make it a final class variable.

**Review by aviaryan — CHANGES_REQUESTED** [review]

File: app/src/main/java/in/aviaryan/hinix/GameBoard.java (hunk 1 above)
Comment (darkwizz) [reply app/src/main/java/in/aviaryan/hinix/GameBoard.java]: done

**Review by aviaryan — APPROVED** [review]

**Comment by aviaryan** [comment]
@darkwizz The app crashes when I run it. Please fix this issue.

**Review by aviaryan — CHANGES_REQUESTED** [review]
The app crashes when I run it. Please fix this issue.

**Comment by darkwizz** [comment]
ok

**Comment by darkwizz** [comment]
fixed

**Pull request closed without merging**